src/components/molecules/Visualizer/hooks.ts

```diff
@@ -211,6 +242,7 @@ export default ({
     innerCamera,
     infobox,
     overriddenSceneProperty,
+    shouldRender: hasTimelineBuiltinWidget,
```

Comment:
I'd like @rot1024's input since I am not fully sure on this, but I feel like this is very specific to Cesium and not sure it belongs here.